src/main/resources/application.properties

```diff
@@ -11,3 +11,8 @@ spring.data.mongodb.auto-index-creation=false
 spring.redis.host=${REDIS_HOST:localhost}
 spring.redis.port=${REDIS_PORT:6379}
 spring.redis.database=${REDIS_DATABASE:0}
+
+#ETHEREUM
+web3j.client-address=${ETHEREUM_NODE_ADDRESS}
```

Comment:
also needs to be configured through `watcher.ethereum.*` prefix.

Reply:
you mean node address?